platform-includes/crons/setup/dotnet.mdx

```diff
@@ -0,0 +1,20 @@
+## Check-Ins (Recommended)
+
+Check-in monitoring allows you to track a job's progress by completing two check-ins: one at the start of your job and another at the end of your job. This two-step process allows Sentry to notify you if your job didn't start when expected (missed) or if it exceeded its maximum runtime (failed).
```

Comment:
How can we bring Hangfire and other integrations to this? I imagine most ppl (I surely did) use a lib like such instead of doing things manually

Reply:
I added a note right after the explainer, pointing towards the Hangfire integration.